app/views/access_limit/edit.html.erb

```diff
@@ -16,6 +16,15 @@
       <%= render "govuk_publishing_components/components/radio", {
         name: "limit_type",
         items: [
+          {
+            value: "none",
+            text: t("access_limit.edit.no_access_limit"),
+            checked: @edition.access_limit.nil?,
```

Comment:
This won't show correct value if a user has to re-submit the form, I assume we don't have requirements yet so can't currently manifest.

Reply:
This is familiar :-). I think it would be better to cover this in the requirements spec - that it replays the set value correctly - when it comes along. How does that sound?

Reply:
Yup, no problem.